doc/src/guides/solving.md

```diff
@@ -0,0 +1,45 @@
+|--------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------|
+| [ Solve an equation algebraically ](http://docs.sympy.org)                        | [  $x^2 = 4 $  ](http://docs.sympy.org)                                                                                                       |
+| [ Solve a system of equations algebraically ](http://docs.sympy.org)              | [  $x^2 + y = 2, x - y = 4$  ](http://docs.sympy.org)                                                                                              |
+| [ Solve an equation numerically ](http://docs.sympy.org)                          | [ $cos(x) = x $ ](http://docs.sympy.org)                                                                                                     |
+| [ Solve a partial differential equation algebraically ](http://docs.sympy.org)    | [${\frac {\partial ^{2}(x^3 y^3)}{\partial x\partial y}}=9 $](http://docs.sympy.org)                                                                     |
+| [ Solve an ordinary differential equation algebraically  ](http://docs.sympy.org) | [ $f\prime\prime(x) + 9f(x)=0 $ ](http://docs.sympy.org)                                                    |
+| [ Solve a system of linear equations algebraically ](http://docs.sympy.org)       | [  $x + y = 2, x - y = 0$  ](http://docs.sympy.org)                                                                                           |
+| [ Solve a system of nonlinear equations algebraically ](http://docs.sympy.org)    | [  $x^2 + y^3 = 1, x^3 - y^2 = 0 $  ](http://docs.sympy.org)                                                                                      |
+| [ Solve a matrix problem algebraically ](http://docs.sympy.org)                   | [ $ \left[\begin{array}{cc} 1 & 1\\1 & -1\end{array}\right] \left[\begin{array}{cc} x\\y\end{array}\right] = \left[\begin{array}{cc} 2\\0\end{array}\right] $ ](http://docs.sympy.org) |
+| [ Solve an inequality algebraically ](http://docs.sympy.org)                      | [ $ x^2 < 4 $ ](http://docs.sympy.org)                                                                                                        |
+| [ Solve a system of inequalities algebraically ](http://docs.sympy.org)           | [ $ x^2 < 4, x > 0 $ ](http://docs.sympy.org)                                                                                                 |
+| [ Solve a polynomial algebraically ](http://docs.sympy.org)                       | [ $ x^2 - x = 0 $ ](http://docs.sympy.org)                                                                                                |
+| [ Solve a diophantine equation algebraically ](http://docs.sympy.org)             | [ $x^2 - 4xy + 8y^2 - 3x + 7y - 5 = 0 $ ](http://docs.sympy.org)                                                                                 |
```

Comment:
It might be more intuitive to describe this as "find integer solutions to an equation".

Reply:
Agree. People might not know what "Diophantine" means, although it is a good idea to tell them since that is the term used in SymPy.

Reply:
Here are the [search queries in the top 1000](https://docs.google.com/spreadsheets/d/1zuQDU1Mj8Ua8zAAxRu5hr7b23wCRlIJ2cOK6FPMnJq0/edit?usp=sharing) involving _diophantine_ or _integer_:

Top queries | Clicks | Impressions | Click-through rate | Position
-- | -- | -- | -- | --
_diophantine_ equation python | 235 | 662 | 35.50% | 1.06
sympy _integer_ | 222 | 387 | 57.36% | 1.1
sympy _diophantine_ | 193 | 292 | 66.10% | 1
python _diophantine_ equation | 63 | 165 | 38.18% | 1.07
_diophantine_ equation solver python | 53 | 131 | 40.46% | 1.01

In total, there are 1250+ searches a year (3.4+ per day) involving _diophantine_, so I think it's well worth including that term here. Here's a revised description to tell visitors what Diophantine means:

`Solve a Diophantine equation (find integer solutions to a polynomial equation) algebraically`